src/erl/erlang/erlang.util.Date.erl

```diff
@@ -0,0 +1,29 @@
+-module('erlang.util.Date').
+
+-include("clojerl.hrl").
+
+-behavior('clojerl.IHash').
+-behavior('clojerl.IEquiv').
+
+-export([?CONSTRUCTOR/1, to_erl/1]).
+
+-export([hash/1]).
+-export([equiv/2]).
+
+-type type() :: #?TYPE{data :: calendar:datetime()}.
+
+-spec ?CONSTRUCTOR(calendar:datetime()) -> type().
+?CONSTRUCTOR(Date) -> #?TYPE{data = Date}.
+
+to_erl(#?TYPE{data = Date}) ->
+    Date.
+
+%% ------------------------------------------------------------------------------
+%% Protocols
+%% ------------------------------------------------------------------------------
+
+hash(Str) ->
```

Comment:
It would be nice to name this argument `Date`. I think this is wrong in `erlang.util.UUID` as well, the product of a copy/paste of mine 😄.